ncm-metaconfig/src/main/metaconfig/opennebula/pan/schema.pan

```diff
@@ -0,0 +1,16 @@
+declaration template metaconfig/opennebula/schema;
+
+include 'pan/types';
+
+type aii_section = {
+    "password" : string
+    "host" : string
+    "port" ? long
+    "user" ? string
+    "pattern" ? string
+} = dict();
+
+@documentation{
+aii/opennebula.conf sections
+}
+type aii_opennebula_conf = aii_section{};
```

Comment:
@kwaegema I'm not sure about this, `Config::Tiny` module is waiting for a  `dict` not an array of dictionaries, at least that what it says the example `src/main/perl/metaconfig.pod` but maybe I'm wrong. In any case this schema is working :smile: but let me know if I have to change something else...